src/ethereum_test_types/types.py

```diff
@@ -335,6 +335,18 @@ class Withdrawal(WithdrawalGeneric[HexNumber]):
 
 
 DEFAULT_BASE_FEE = 7
+CURRENT_MAINNET_BLOCK_GAS_LIMIT = 36_000_000
+DEFAULT_BLOCK_GAS_LIMIT = CURRENT_MAINNET_BLOCK_GAS_LIMIT * 2
```

Comment:
I'm thinking this should be a sane limit of always filling the tests with twice the value we currently have in mainnet, with the possibility of preemptively bumping it before we signal a safe increment to client devs/node operators.